src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -1,11 +1,19 @@
-float calcLightAttenuation( const in float lightDistance, const in float cutoffDistance, const in float decayExponent ) {
+// convert from intensity (power / solid angle) to irradiance assuming a spherical light falloff
+// this is only valid for spot and point lights
+// physically correct inverse quadratic falloff is achieved when decayExponent is 2.
+// discussed here: https://github.com/mrdoob/three.js/issues/7608
+float intensityToIrradianceSphericalFalloffCoefficient( const in float lightDistance, const in float cutoffDistance, const in float decayExponent ) {
 
 	if ( decayExponent > 0.0 ) {
 
-	  return pow( saturate( -lightDistance / cutoffDistance + 1.0 ), decayExponent );
+		float distanceFalloff = 1.0 / pow( max( lightDistance, EPSILON ), decayExponent );
```

Comment:
What you say is true, but what concerns me is that users who create a scene of molecules, and users who create a galaxy, likely will not have distance units of meters. This is why the current falloff calculation is unit-less.

I agree with you that a physical falloff model is best, but I do not know how to implement it and accommodate all users -- unless we convert to meters from user-specified distance units.

I do understand that you feel it will be OK to tell users that their models must be in meters, so I will defer to whatever you and @mrdoob decide.